astronomer/providers/amazon/aws/example_dags/example_sagemaker.py

```diff
@@ -0,0 +1,502 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Remove license

Reply:
@pankajastro we are working on the example DAG for now to create it as end to end, please ignore for now